docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Serial Vault User Guide
+table_of_contents: true
+---
+
+# Serial Vault user guide
+
+The Serial Vault is a Canonical-hosted web service that is required to use a
+[Secure Brand Store][secure-brand-store]. The Serial Vault provides :
+
+ - Signed [serial assertions][serial-assertions]  to authenticated systems of
+   a given brand/model, which then use it to access a secure Brand Store.
```

Comment:
Should we always capitalize "Secure Brand Store" or always not have "secure" be capitalized? We just need to make sure to use it consistently in this doc.

Reply:
Since all brand stores are (or should be secure) I don't think Secure should be capitalized (unless for other obvious grammatical reasons.